prow/plank/controller.go

```diff
@@ -439,7 +436,7 @@ func (c *Controller) syncTriggeredJob(pj kube.ProwJob, pm map[string]kube.Pod, r
 	}
 
 	if pj.Status.State == kube.TriggeredState {
-		// BuildID needs to be set before we execute the job url template.
+		// ProwJobID needs to be set before we execute the job url template.
```

Comment:
`ProwJobID` is not set.

Reply:
Whoops, bad refactor rename button in IDEA